studio/components/interfaces/Organization/BillingSettingsV2/Subscription/Subscription.tsx

```diff
@@ -78,6 +93,32 @@ const Subscription = () => {
                 <p className="text-2xl text-brand uppercase">{currentPlan?.name ?? 'Unknown'}</p>
               </div>
 
+              {subscription?.scheduled_plan_change &&
+                subscription?.scheduled_plan_change?.target_plan !== subscription.plan.id && (
+                  <Alert_Shadcn_ className="mb-2">
+                    <IconCalendar className="h-4 w-4" />
+                    <AlertDescription_Shadcn_ className="flex justify-between items-center">
```

Comment:
suspicious classNames...

Reply:
<img width="822" alt="Screenshot 2023-10-23 at 11 52 31" src="https://github.com/supabase/supabase/assets/14073399/02ea71f7-df73-4a4c-87d1-879119b86993">

here's what I'm trying to do - calendar icon, bit of text and a CTA on the right - couldn't quickly find an example with the shadcn alert for this

Reply:
Pushed up some changes after talking to Joshen

Reply:
is it actually an alert ? 
the text seems to say it is.

Just because alert uses role="alert"
https://github.com/supabase/supabase/blob/master/packages/ui/src/components/shadcn/ui/alert.tsx/#L33

Reply:
yea that makes sense.